website/content/en/docs/building-operators/golang/quickstart.md

```diff
@@ -87,6 +87,7 @@ Read the [tutorial][tutorial] for an in-depth walkthough of building a Go operat
 [go_tool]:https://golang.org/dl/
 [docker_tool]:https://docs.docker.com/install/
 [kubectl_tool]:https://kubernetes.io/docs/tasks/tools/install-kubectl/
-[operator_install]: /docs/installation/install-operator-sdk
+[operator_install]: /docs/installation/
+[envtest-setup]: /docs/building-operators/golang/references/envtest-setup
```

Comment:
I don't see this link being used anywhere
```suggestion
```

Reply:
The element this links to got removed in [3983](https://github.com/operator-framework/operator-sdk/pull/3983), I'll clean up the trailing link in my PR as well.